After honcho 0.6 port should be a multiple of 1000

I used honcho for a while and liked it ability to use custom base ports, so its no need to start base ports from multiple of 1000 as in foreman. 

But after 0.6 release this changed and now I got something like:

```
PORT=8110 . venv/bin/activate && honcho start web -f ec2Procfile
. venv/bin/activate && honcho start worker
Traceback (most recent call last):
  File "/home/playpauseandstop/Projects/.../.../venv/bin/honcho", line 11, in <module>
    sys.exit(main())
  File "/home/playpauseandstop/Projects/.../.../venv/lib/python2.7/site-packages/honcho/command.py", line 269, in main
    COMMANDS[args.command](args)
  File "/home/playpauseandstop/Projects/.../.../venv/lib/python2.7/site-packages/honcho/command.py", line 212, in command_start
    port=port):
  File "/home/playpauseandstop/Projects/.../.../venv/lib/python2.7/site-packages/honcho/environ.py", line 114, in expand_processes
    assert port % 1000 == 0, "port must be multiple of 1000"
AssertionError: port must be multiple of 1000
```

when try to start honcho on ``8110`` port. 

Do we really need mimicrify foreman here? Or can continue live with custom base ports?

--- a/honcho/environ.py
+++ b/honcho/environ.py
@@ -110,9 +110,6 @@ def expand_processes(processes, concurrency=None, env=None, quiet=None, port=Non
     if env is not None and env.get("PORT") is not None:
         port = int(env.get("PORT"))
 
-    if port is not None:
-        assert port % 1000 == 0, "port must be multiple of 1000"
-
     if quiet is None:
         quiet = []
 

--- a/honcho/test/unit/test_environ.py
+++ b/honcho/test/unit/test_environ.py
@@ -206,9 +206,6 @@ class TestExpandProcesses(TestCase):
         p = ep(("foo", "some command"), port=8000)
         self.assertEqual({"PORT": "8000"}, p[0].env)
 
-    def test_port_invalid(self):
-        self.assertRaises(AssertionError, ep, ("foo", "test"), port=5123)
-
     def test_port_multiple(self):
         p = ep(("foo", "some command"),
                ("bar", "another command"),

